schemas/src/actions/Action.Execute.json

```diff
@@ -0,0 +1,17 @@
+{
+	"$schema": "https://raw.githubusercontent.com/microsoft/AdaptiveCards/6f39aedce45864ae1067ed44a5551dc973790bb5/source/nodejs/typed-schema/schema/lib/Type.json",
+	"extends": "Action, ISelectAction",
```

Comment:
just to be sure -- we want `Action.Execute` to be a `selectAction` target, yes?

Reply:
Yes